ros_ws/src/get_user_request.py

```diff
@@ -0,0 +1,141 @@
+import os
+import langchain
+
+from langchain.llms import OpenAI
+from langchain import PromptTemplate
+from langchain import schema
+from langchain.prompts import (ChatPromptTemplate,PromptTemplate,SystemMessagePromptTemplate, HumanMessagePromptTemplate)
+from langchain.chat_models import ChatOpenAI
+from langchain.chains import LLMChain
+from langchain.schema import HumanMessage, SystemMessage, AIMessage
```

Comment:
Run `isort` to fix import. In fact, in general run linting on this via the `just` command I setup for that, and/or set up linting on save by default in vs code to make your life a bit easier.

Reply:
isort run